.github/workflows/ci-cd-staging.yml

```
@@ -83,6 +83,17 @@ jobs:
       version: ${{ needs.get-current-version.outputs.version }}
       runMigration: ${{ github.event_name == 'workflow_dispatch' || needs.check-for-changes.outputs.hasMigrationChanges == 'true' }}
 
+  publish-sdk-to-nuget:
+    uses: ./.github/workflows/workflow-publish-nuget.yml
+    needs: [ get-current-version, check-for-changes ]
+    if: ${{ needs.check-for-changes.outputs.hasBackendChanges == 'true' || needs.check-for-changes.outputs.hasTestChanges == 'true' }}
```

Nei, backend/test skal ikke trigge dette?

@arealmaas Skal ikke denne bare være 

```suggestion
    if: ${{ needs.check-for-changes.outputs.hasSchemaChanges == 'true' }}
```

? 🤔

---

Du har helt rett!